gpMgmt/test/behave/mgmt_utils/gpinitsystem.feature

```diff
@@ -17,6 +17,11 @@ Feature: gpinitsystem tests
         And gpconfig should print "Master  value: off" to stdout
         And gpconfig should print "Segment value: off" to stdout
 
+	Scenario: gpinitsystem exits with status 0 when the user set locale parameters 
```

Comment:
It seems that spaces should be used for indentation instead for tabs.

Reply:
Trailing space too